### ISSUE TITLE
specs fail randomly with Process pause/resume feature

### ISSUE THREAD

**Issue by Fryguy** [ManageIQ/more_core_extensions#84]
While #82 should have fixed this, it seems it's not working, as evidenced in #83 .

@djberg96 Can you look into this?

**Comment by djberg96** [comment]
Aargh! I think maybe it needs a full second, since I think that's how often the proc filesystem gets updated on Linux.

**Comment by djberg96** [comment]
I don't get it, even with no delay, I can't duplicate this on my local Linux vm:

```
require 'sys-proctable'

pid = fork

if pid
  p Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false).state # S
  Process.kill('STOP', pid)
  p Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false).state # T
  Process.kill('CONT', pid)
  p Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false).state # S
  Process.kill('KILL', pid)
  p Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false).state # Z
else
  sleep 100
end
```

**Comment by djberg96** [comment]
Replaced with a straight `ps` call.

Note that I can replicate the same issue with `MiqProcess.linux_process_stat`, so it's not sys-proctable per se.

**Comment by Fryguy** [comment]
Closed via #85
